proposal/comp110_architecture_proposal.tex

```diff
@@ -17,58 +17,58 @@ \section*{Topic}
+\item[Comments:]  This article attempts to demonstrate how level creation should be generatored through functions that compare the intended difficultly by the desinger and the difficulty for the player content.
 \end{description}
 
 \section*{Paper 3}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] PCG-based game design: creating Endless Web
+\item[Citation:] \cite{Smith:2012}
+\item[Abstract:] "This paper describes the creation of the game Endless Web, a 2D platforming game in which the player's actions determine the ongoing creation of the world she is exploring. Endless Web is an example of a PCG-based game: it uses procedural content generation (PCG) as a mechanic, and its PCG system, Launchpad, greatly influenced the aesthetics of the game. All of the player's strategies for the game revolve around the use of procedural content generation. Many design challenges were encountered in the design and creation of Endless Web, for both the game and modifications that had to be made to Launchpad. These challenges arise largely from a loss of fine-grained control over the player's experience; instead of being able to carefully craft each element the player can interact with, the designer must instead craft algorithms to produce a range of content the player might experience. In this paper we provide a definition of PCG-based game design and describe the challenges faced in creating a PCG-based game. We offer our solutions, which impacted both the game and the underlying level generator, and identify issues which may be particularly important as this area matures."
+\item[Web link:] \url{http://dl.acm.org/citation.cfm?id=2282338.2282375&coll=DL&dl=GUIDE&CFID=729021823&CFTOKEN=29015746}
+\item[Full text link:] \url{https://games.soe.ucsc.edu/sites/default/files/smith-fdg12.pdf}
+\item[Comments:] This is article suggests the influence the player's actions have on an endless 2D platform game. It also reviews it's own generator aswell as stating the challenges of creating a PCG-based game.
 \end{description}
 
 \section*{Paper 4}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] Polymorph: A Model for Dynamic Level Generation
```

Comment:
This paper is also based on Smith et al's work. However the addition of dynamic difficulty adjustment may be enough to justify reviewing it separately.